test cases/linuxlike/15 ld binary/meson.build

```diff
@@ -0,0 +1,4 @@
+project('ld binary')
+
+ld = find_program('ld')
+assert(run_command(ld, '--version').returncode() == 0)
```

Comment:
I just realized that the cross files in `cross` subdirectory should have ld entries too. Maybe add them to armhf and mingw32 and leave it out from mingw64 so that both cases get tested?

Reply:
done. I'm waiting for my windows machine to finish running updates, then I'll try to sort those out.